Document instructions for building SDK in non SGX Windows Machines (#3158)
openenclave/openenclave: docs/GettingStartedDocs/Contributors/SimulatorGettingStartedWindows.md

> # Getting Started with Open Enclave in Simulation mode on Windows Machines Without SGX Support.

Reviewer: ```suggestion
# Getting Started with Open Enclave in Simulation Mode on Windows Machines Without SGX Support
```

## Platform requirement

Intel® X86-64bit architecture

A version of Windows OS :
- For server: Windows Server 2016 or 2019
- For client: Windows 10 64-bit version 1709 or newer
- To check your Windows version, run `winver` from the command line

> *Note:* The following instructions assume running `powershell` as adminstrator.

Reviewer: ```suggestion
*Note:* The following instructions assume running `powershell` as administrator.
```

- Clone the Open Enclave SDK to a folder of your choice. In these instructions
  we're assuming `C:/Users/test`.

```powershell
cd C:/Users/test/
git clone --recursive https://github.com/openenclave/openenclave.git
```

This creates a source tree under the directory called `openenclave`.

## Install project prerequisites

First, change directory into the Open Enclave repository (from wherever you
cloned it):

```powershell
cd C:/Users/test/openenclave
```

Also, make sure the execution policy is set to `RemoteSigned` with the following command.

```powershell
Get-ExecutionPolicy
```

If not, set the policy with the following command and confirm the change by typing `Y`.

```powershell
Set-ExecutionPolicy RemoteSigned
```

To deploy all the prerequisities for building Open Enclave, you can run the
following from PowerShell:

```powershell
./scripts/install-windows-prereqs.ps1 -InstallPath C:/oe_prereqs -LaunchConfiguration SGX1-NoDriver -DCAPClientType None
```

## Building on Windows using Developer Command Prompt

Launch the [x64 Native Tools Command Prompt for VS(2017 or 2019)](
https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs),

> which is found in the `Visual Studio 2017` folder in the Start Menu.

Reviewer: ```suggestion
which is found in the `Visual Studio 2017` (or 2019) folder in the Start Menu.
```

The following tests did not run:
         51 - tests/attestation_cert_api (Skipped)
         52 - tests/attestation_plugin (Skipped)
         54 - tests/bigmalloc (Skipped)
         59 - tests/debug-unsigned (Skipped)
         60 - tests/debug-signed (Skipped)
         61 - tests/nodebug-signed (Skipped)
         62 - tests/nodebug-unsigned (Skipped)
        197 - tests/qeidentity (Skipped)
        198 - tests/report (Skipped)
        199 - tests/report_attestation_without_enclave (Skipped)
        202 - tests/sealKey (Skipped)
        207 - tests/global_init_exception (Skipped)
        220 - tests/VectorException (Skipped)
```

Some of the tests are skipped (Not Run) by design because the current simulator is not fully featured yet.

A clean pass of the above unit tests is an indication that your Open Enclave setup was successful.

You can start playing with those Open Enclave samples after following the instructions in the "Install" section below to configure samples for building,

For more information refer to the [Advanced Test Info](AdvancedTestInfo.md) document.

## Install

The `ninja install` command will install the SDK in `C:/openenclave`

## Build and run samples

> To build and run the samples, please look [here](/samples/README_Windows.md).

Reviewer: ```suggestion
To build and run the samples, see the [Windows samples README file](/samples/README_Windows.md).
```
